src/main/java/io/github/sskorol/assertions/LoginPageAssert.java

```diff
@@ -1,29 +1,30 @@
 package io.github.sskorol.assertions;
 
-import io.github.sskorol.model.Account;
+import io.github.sskorol.pages.LoginPage;
 import io.qameta.allure.Step;
 import org.assertj.core.api.AbstractAssert;
 import org.assertj.core.util.Objects;
 
 /**
  * A simple model class for Login Page assert processing.
  */
-public class LoginPageAssert extends AbstractAssert<LoginPageAssert, Account> {
+public class LoginPageAssert extends AbstractAssert<LoginPageAssert, LoginPage> {
 
     private static final String ERROR_MESSAGE_TEMPLATE = "\nExpecting <%s> of: \n<%s>\nto be: <%s>\nbut was: <%s>\n";
 
-    public LoginPageAssert(final Account account) {
-        super(account, LoginPageAssert.class);
+    public LoginPageAssert(final LoginPage actual) {
```

Comment:
Rename "actual" to "loginPage" for avoiding misunderstanding. Because actual - it's protected final ACTUAL actual from AbstractAssert class.